x-pack/plugins/ingest_pipelines/public/application/sections/manage_processors/add_database_modal.tsx

```diff
@@ -0,0 +1,265 @@
+  };
+  const onAddDatabase = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (!isFormValid()) {
+      return;
+    }
+    setIsLoading(true);
+    try {
+      const { error } = await services.api.createGeoipDatabase({
+        databaseType,
+        databaseName,
+        maxmind,
+      });
+      setIsLoading(false);
+      if (error) {
+        services.notifications.toasts.addError(error, {
+          title: addDatabaseErrorTitle,
+        });
+      } else {
+        services.notifications.toasts.addSuccess(getAddDatabaseSuccessMessage(databaseName));
+        await reloadDatabases();
+        closeModal();
+      }
+    } catch (e) {
+      setIsLoading(false);
+      services.notifications.toasts.addError(e, {
+        title: addDatabaseErrorTitle,
+      });
+    }
+  };
+
+  return (
+    <EuiModal
+      css={css`
+        width: 500px;
+      `}
+      aria-labelledby={ADD_DATABASE_MODAL_TITLE_ID}
+      onClose={closeModal}
+      initialFocus={'[data-test-subj="datanaseTypeSelect"]'}
+    >
+      <EuiModalHeader>
+        <EuiModalHeaderTitle id={ADD_DATABASE_MODAL_TITLE_ID}>
+          <FormattedMessage
+            id="xpack.ingestPipelines.manageProcessors.geoip.addDatabaseModalTitle"
+            defaultMessage="Add database"
+          />
+        </EuiModalHeaderTitle>
+      </EuiModalHeader>
+
+      <EuiModalBody>
+        <EuiForm
+          fullWidth={true}
+          id={ADD_DATABASE_MODAL_FORM_ID}
+          component="form"
+          onSubmit={(event) => onAddDatabase(event)}
+        >
+          <EuiFormRow
+            label={
+              <FormattedMessage
+                id="xpack.ingestPipelines.manageProcessors.geoip.addDatabaseForm.databaseTypeSelectLabel"
+                defaultMessage="Type"
+              />
+            }
+          >
+            <EuiSelect
+              options={DATABASE_TYPE_OPTIONS}
+              hasNoInitialSelection={true}
+              value={databaseType}
+              onChange={(e) => onDatabaseTypeChange(e.target.value)}
+              data-test-subj="datanaseTypeSelect"
+            />
+          </EuiFormRow>
+          {databaseType && (
+            <>
+              <EuiSpacer />
+              <EuiCallOut
+                title={
+                  <FormattedMessage
+                    id="xpack.ingestPipelines.manageProcessors.geoip.licenseCalloutTitle"
+                    defaultMessage="Add MaxMind license key to keystore"
+                  />
+                }
+                iconType="iInCircle"
+              >
+                <p>
+                  <FormattedMessage
+                    id="xpack.ingestPipelines.manageProcessors.geoip.licenseCalloutText"
+                    defaultMessage="In order to grant access to your MaxMind account, you must add the license key to the keystore. {link}"
+                    values={{
+                      link: (
+                        <EuiLink href="#">
```

Comment:
Is this doc link still not available or was it just missed?

Reply:
Still waiting on confirmation about this one 😬